source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -193,3 +193,19 @@ For example:
     })
   </script>
 ```
+
+### If your JavaScript isn’t working properly
+
+If your site has a Content Security Policy (CSP), the CSP may block the inline JavaScript in the page template. You may see a warning similar to the following in the developer tools in your browser:
+
+```
+Refused to execute inline script because it violates the following Content Security Policy directive: "default-src 'self'".
+```
+
+You can unblock the JavaScript by asking your security specialist to include the following hash in your CSP:
```

Comment:
Is "security specialist" a bit too prescriptive here? Just thinking about organisations/projects where the CSP is owned by developers, this might be a bit confusing?

Reply:
+1 – I think the intention here is to make sure people who care about this are involved when the change is made? Definitely agree with that, but as it reads at the minute I think I'm not sure it'd mean anything to most people. And even if there is someone 'overseeing' the use of CSP etc, it's likely that the actual change would be made by developers.

Reply:
Good point 👍 

> I think the intention here is to make sure people who care about this are involved when the change is made?

Yes. The changes we usually require users to make are localised to a component or pages but changing your CSP could impact your whole app if you get it wrong. So it might be useful to ask users to take special care here and/or flag that this is something that we wouldn't necessarily expect the developer to do on their own.

What do you think? (We can always ask for help to word this in a better way)

Reply:
Resolved! After speaking to @m-green we've now removed the 'security specialist' mention but link to some CSP guidance.